roles/ssp-modules/tasks/install-common.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+
+- name: Checkout SSP modules source
+  git:
+    repo: "{{ item.url }}"
+    dest: "{{ item.path }}"
+    version: "{{ item.version }}"
+    accept_hostkey: yes
+    force: no
+    update: yes
+  become: yes
+  with_items: "{{ ssp_modules }}"
+
+
+- name: Create PostgreSQL tables for SSP modules via SQL scripts
+  postgresql_query:
+    db: "{{ item.db_name }}"
+    login_user: "{{ item.db_username }}"
+    login_password: "{{ item.db_password }}"
+    path_to_script: "{{ item.db_script_path }}"
+    login_host: "{{ item.db_host }}"
+    #positional_args:
+    #- 1
+  register: postgresql_result
+  failed_when: "postgresql_result is failed and 'already exists' not in postgresql_result.msg"
+  become: yes
```

Comment:
check if this is necessary

Reply:
Yes it is so that we can run the same .sql script over and over again ( [for example](https://github.com/rciam/proxystatistics-simplesamlphp-module/blob/rciam-3.0/config-templates/pgsql_tables.sql) ).

Reply:
`become: yes` is not necessary.